src/main/java/uk/gov/homeoffice/digital/sas/balancecalculator/models/accrual/enums/AccrualType.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.homeoffice.digital.sas.balancecalculator.models.accrual.enums;
+
+import java.util.UUID;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public enum AccrualType {
+  ANNUAL_TARGET_HOURS(UUID.fromString("e502eebb-4663-4e5b-9445-9a20441c18d9"), true),
+  NIGHT_HOURS(UUID.fromString("5f06e6ce-1422-4a0c-89dd-f4952e735202"), false),
```

Comment:
Should these be omitted until each one is introduced? Alternatively have each perform a no-op rather than enabled or not

Reply:
This is exactly what we tried to achieve with the `enabled` attribute. Having them present would spare us the inconvenience of having to lookup the accrual type id in accruals project and copy them across  every time we introduce individual accrual type